www/frontend/index.html

```diff
@@ -0,0 +1,129 @@
+                        li.setAttribute('id', 'taskItem_'+data.message);
+                        li.appendChild(document.createTextNode(data.message));
+                        let deleteButton = document.createElement("button");
+                        deleteButton.setAttribute('id', 'task_'+ data.message);
+                        deleteButton.setAttribute('onclick','deleteTask();'); // for FF
+                        deleteButton.onclick = function() { deleteTask(data.message); }; // for IE
+                        deleteButton.innerText = "Delete";
+
+                        let updateButton = document.createElement("button");
+                        updateButton.setAttribute('id', 'task_'+ data.message);
+                        updateButton.setAttribute('onclick','updateTask();'); // for FF
+                        updateButton.onclick = function() { updateTask(data.message); }; // for IE
+                        updateButton.innerText = "Update";
+
+                        li.append(deleteButton);
+                        li.append(updateButton);
+                        ul.append(li);  
+                    },
+                    failure: function(error) {
+                        alert("this is an error" + error);
+                    }
+                });
+            });
+           
+            function getTask() {
+               
+                $.ajax({
+                    method: "GET",
+                    url: "http://localhost/backend-api/router.php",
+                    success: function(data) {
+                      alert(data);
+
+                    },
+                    failure: function(error) {
+                        alert("This is an error: " + error);
+                    }
+                });
+            }
+
+            function updateTask(task) {
+                let newTask = prompt('Insert new task');
+                const data = JSON.stringify({
+                    oldTask: task,
+                    newTask: newTask
+                });
+                $.ajax({
+                    async: true,
+                    crossDomain: true,
+                    contentType: 'application/json; charet=utf-8',
+                    data: data,
+                    processData: false,
+                    method: "PUT",
+                    url: "http://localhost/backend-api/router.php",
+                    success: function(data) {
+                        let itemToBeUpdated = document.getElementById('taskItem_' + task);
+                        textNode = itemToBeUpdated.childNodes[0];
+                        textNode.nodeValue = newTask;
+                    },
+                    failure: function(error) {
+                        alert("This is an error: " + error);
+                    }
+                });
+            }
+
+            function deleteTask(task) {
+                const data = JSON.stringify({
+                    task: task
```

Comment:
Any reason why the server needs anything other than the task id to complete the deletion of a task? This just seems to make for a larger request body.